cloud-service-providers/google-cloud/gke/variables.tf

```diff
@@ -390,11 +396,11 @@ variable "vm_gpu_spec_list" {
   }
 }
 
-variable "region_based_vm" {
-  type        = string
-  description = "Cluster and GPU location"
-  default     = "L4 us-east4 g2-standard-24"
-}
+#variable "region_based_vm" {
```

Comment:
I'd rather not have commented out variables in the checked in code. If this isn't being used, please remove it.